cns/cniconflist/generator_linux.go

```diff
@@ -133,3 +133,31 @@ func (v *CiliumGenerator) Generate() error {
 
 	return nil
 }
+
+// Generate writes the CNI conflist to the Generator's output stream
+func (v *SWIFTGenerator) Generate() error {
+	conflist := cniConflist{
+		CNIVersion: overlaycniVersion,
```

Comment:
not overlay/swift specific so we should change this variable to `azurecniVersion`